packages/SwingSet/src/controller/controller.js

```diff
@@ -216,7 +217,7 @@ export async function makeSwingsetController(
       if (finished) {
         // `finish` should only be called once.
         // Log a stack-bearing error instance, but throw something more opaque.
-        const msg = `slog event ${startLabel} ${q(startProps || {})} already finished; ignoring props ${q(extraProps || {})}`;
+        const msg = `slog event ${q(startLabel)} ${q(startProps || {})} already finished; ignoring props ${q(extraProps || {})}`;
```

Comment:
Making `startLabel` possible to be `undefined` seems to imply that `endLabel` is sometimes a better choice for this log message:


```suggestion
        const msg = `slog event ${q(startLabel || endLabel)} ${q(startProps || {})} already finished; ignoring props ${q(extraProps || {})}`;
```